### PR TITLE
Resolving issue 74 - TypeError: Cannot read property 'login' of null

### PR DESCRIPTION
**Pull request by maoo**
I managed to catch when a PR contains unidentifiable authors, which are all those GitHub users who haven't properly set their GitHub email address and/or `git config email.address` local setup.

I've added a new configuration in the `.clabot` that defaults to the following:
```
  "messageMissingEmail": "Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **{{unidentifiedUsers}}**.\nThis is most likely caused by a git client misconfiguration; please make sure to:\n1. check if your git client is configured with an email to sign commits `git config --list | grep email`\n2. If not, set it up using `git config --global user.email email@example.com`\n3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails",
```

I've changed the `index.js` and implemented lambda specs to test the specific corner case, `npm test` is happy. I've also tested it on our staging account, see screenshot below.

screenshot 2018-11-22 at 22 54 33

Eager to hear your thoughts. Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "execute:bot": "serverless invoke local --function cla-bot --path cla-bot/dev/event.json --stage dev",
+    "execute:bot": "serverless invoke local --function cla-bot --path src/dev/event.json --stage dev",
```
Comment (ColinEberhardt) [inline package.json:7]: Aha - well spotted!

**Review by ColinEberhardt — APPROVED** [review]

**Comment by ColinEberhardt** [comment]
Thanks @maoo - that's a really useful new feature
